app/assets/stylesheets/article-show.scss

```diff
@@ -95,6 +95,7 @@ header{
     z-index:5;
   }
   .title{
+    z-index:6;
```

Comment:
Might be dangerous?

Reply:
Should be fine if it passes muster in your own browser, but good instincts in terms of thinking that it's sort of dangerous.